<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-KKG1FK9R" failures="1" tests="7" name="Default test" time="35.414" errors="0" timestamp="2020-01-30T09:42:38 TST">
  <testcase classname="demoJenkins.LoginValidation" name="test9" time="0.0">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;account&amp;apos; is required by @Test on method test9 but has not been marked @Optional or defined
in C:\Users\user\AppData\Local\Temp\testng-eclipse--1151542503\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'account' is required by @Test on method test9 but has not been marked @Optional or defined
in C:\Users\user\AppData\Local\Temp\testng-eclipse--1151542503\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:289)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:377)
at org.testng.internal.Parameters.createParameters(Parameters.java:721)
at org.testng.internal.Parameters.handleParameters(Parameters.java:895)
at org.testng.internal.Parameters.handleParameters(Parameters.java:762)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:60)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:39)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:771)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test9 -->
  <testcase classname="demoJenkins.LoginValidation" name="test10" time="1.173"/>
  <testcase classname="demoJenkins.LoginValidation" name="test11" time="0.918"/>
  <testcase classname="demoJenkins.LoginValidation" name="test12" time="0.169"/>
  <testcase classname="demoJenkins.LoginValidation" name="test13" time="1.09"/>
  <testcase classname="demoJenkins.LoginValidation" name="test14" time="15.94"/>
  <testcase classname="demoJenkins.LoginValidation" name="test15" time="5.482"/>
</testsuite> <!-- Default test -->
